blog_post_collaboration/deploy_Kedro_pipelines_on_EMR.md

```diff
@@ -0,0 +1,165 @@
+# Title TBD. Maybe "Seven steps to deploy Kedro pipelines on AWS EMR" ??
+
+## Introduction
+
+
+[Amazon EMR](https://aws.amazon.com/emr/) (previously called Amazon Elastic MapReduce) is a managed cluster platform that simplifies running big data frameworks, such as Apache Hadoop and Apache Spark, to process and analyze vast amounts of data with AWS.
+
+> TO DO 
+This section needs another sentence or two to set the scene on a typical use case of getting Kedro working with EMR.
+
+## 1. Set up the cluster environment
+
+In order to install Python libraries on EMR, packaging a virtual environment and deploying it to EMR is a neat option. 
+
+To create and package a virtual environment the cluster needs to have the [same Amazon Linux 2
+environment as used by EMR](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide).
+
+To find out more about how to package your dependencies on an Amazon Linux 2 base, we used this [example Dockerfile](https://github.com/aws-samples/emr-serverless-samples/tree/main/examples/pyspark/dependencies). Our example Dockerfile is as below: 
+
+```text
+FROM --platform=linux/amd64 amazonlinux:2 AS base 
+
+RUN yum install -y python3 
+
+ENV VIRTUAL_ENV=/opt/venv 
+RUN python3 -m venv $VIRTUAL_ENV 
+ENV PATH="$VIRTUAL_ENV/bin:$PATH" 
+
+COPY requirements.txt /tmp/requirements.txt 
+
+RUN python3 -m pip install --upgrade pip && \
+    python3 -m pip install venv-pack==0.2.0 && \ 
+    python3 -m pip install -r /tmp/requirements.txt 
+    
+RUN mkdir /output && venv-pack -o /output/pyspark_deps.tar.gz 
+
+FROM scratch AS export 
+COPY --from=base /output/pyspark_deps.tar.gz /
+```
+
+
+> Note: A `DOCKER_BUILDKIT` backend is necessary to run this Dockerfile (make sure you have it installed).
+
+Run the Dockerfile  using the following command. This will generate a `pyspark_deps.tar.gz`  file at the  `<output-path>`  specified in the command above. 
+
+```bash
+DOCKER_BUILDKIT=1 docker build --output . <output-path> 
+```
+
+Use this command if your Dockerfile has a different name 
+
+```bash
+DOCKER_BUILDKIT=1 docker build -f Dockerfile-emr-venv --output . <output-path>
+```
+
+## 2. Set up `CONF_ROOT`
+
+By default, Kedro looks at the root `conf` folder for all its configurations (catalog, parameters, globals, credentials, logging) to run the pipelines. However, [this can be customized](https://docs.kedro.org/en/stable/kedro_project_setup/configuration.html#configuration-root) by changing `CONF_ROOT`  in `settings.py`. 
```

Comment:
To make this more annoying in 0.18.7 we do package configuration as a separate `tar.gz` file which can be used in conjunction with then `--conf-source` flag